docs/DeveloperGuide.adoc

```diff
@@ -358,14 +358,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *`   |Forgetful student | Set reminders for upcoming events | Will remember to attend them
 
+|`* * *` |Student | Add tasks to study plan | Can add study tasks to my study plan
+
+|`* * *` |Student | Delete tasks on study plan | Can delete study tasks if I don't need it anymore
+
+|`* * *` |Student | Auto-generate event from a task | If I feel like I want to do a study task today, I can generate an event on today's schedule, or any day.
+
+|`* *`   |Student | View statistics of my task progress | Can see how much I've done for each tasks on my study plan.
```

Comment:
For the last column, do you mean "Can see how many tasks I have completed for my study plan"? I assume that tasks are the smallest units in your study planner so "how much of each task" seems inaccurate.

Reply:
Tasks is a group of events if you will... For example, a task is "I want to study 2 hours per day for 100 hours", and I can see how many events that I've created i.e. how much I've done for the task, something like "You've done 54 hours out of 100 hours"

Reply:
In that case, just remove the 's' for tasks,